metadata-ingestion/src/datahub/ingestion/source/looker/looker_common.py

```diff
@@ -1131,7 +1132,15 @@ def from_api(  # noqa: C901
                 logger.warning(
                     f"Failed to extract explore {explore_name} from model {model}: {e}"
                 )
-
+        except DeserializeError as e:
+            logger.warning(
+                f"Failed to extract explore {explore_name} from model {model}, Error: {e}"
+            )
+            reporter.report_warning(
+                title=f"Failed to extract explore {explore_name} from model {model}",
+                message="Encountered exception while attempting to find dependent views for this chart",
+                context=f"Explore: {explore_name}, Mode: {model}",
```

Comment:
see our internal doc on error reporting conventions / standards here https://www.notion.so/acryldata/Error-reporting-in-ingestion-5343cc6ea0c84633b38070d1a409c569?pvs=4

Reply:
Done, Can you please check again